examples/with-facebook-chat-plugin/.env.local.example

```
@@ -0,0 +1 @@
+FACEBOOK_PAGE_ID=
```

```suggestion
NEXT_PUBLIC_FACEBOOK_PAGE_ID=
```

See: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser